docs/raspberry.md

```diff
@@ -0,0 +1,74 @@
+# Installing on a Raspberry Pi
+
+!!! Warning
+	This process currently only installs the requirements to run the bot live. You will need to do the backtesting and optimising on different (more powerful) hardware.
+
+!!! Info	
+	Tested on a Raspberry Pi 3b with the latest Raspberry Pi OS.
+
+Raspberry Pi OS currently uses Python2.7 and Python3.7, we need Python3.8.x. If you already have python3.8 installed, you can skip ahead to `Installing the requirements`.
+
+## Installing python3.8
+
+To install the dependencies for Python3.8 run the following:
+
+```shell
+sudo apt-get update
+sudo apt-get install -y build-essential tk-dev libncurses5-dev libncursesw5-dev libreadline6-dev libdb5.3-dev libgdbm-dev libsqlite3-dev libssl-dev libbz2-dev libexpat1-dev liblzma-dev zlib1g-dev libffi-dev
+```
+
+To download, unpack and install Python3.8 run the following (this might take a long time):
+
+```shell
+wget https://www.python.org/ftp/python/3.8.0/Python-3.8.0.tgz
```

Comment:
Is there a specific reason for not using the latest 3.8 version of python, but instead the 3.8.0?

Reply:
Not really, I tried with 3.8.9 originally and ran into some issues so thought I'd try with 3.8.0. In retrospect, something else was probably causing the issues. I will test it with the latest 3.8 tomorrow and update if it works